src/webgpu/util/floating_point.ts

```diff
@@ -4317,7 +4356,10 @@ export abstract class FPTraits {
 
   private readonly SubtractionIntervalOp: ScalarPairToIntervalOp = {
     impl: (x: number, y: number): FPInterval => {
-      return this.correctlyRoundedInterval(x - y);
+      const difference: number = x - y;
```

Comment:
This is the addition case    but using -y for the second argument.
It was easier to understand when I read it that way.  May be worth a comment.